scripts/lib/stringUtils.ts

```diff
@@ -0,0 +1,38 @@
+// This code is a Qiskit project.
```

Comment:
While this file is a copy from the internal repo, I removed functions not used by this API pipeline.